libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.ts

```diff
@@ -368,8 +368,6 @@ export class MultiComboboxComponent extends BaseMultiCombobox implements OnInit,
         // selected items should be displayed in the same order as options
         const valueIndexes = new Map<any, number>(this._suggestions.map((s, i) => [s.value, i]));
         this._selectedSuggestions.sort((a, b) => valueIndexes.get(a.value)! - valueIndexes.get(b.value)!);
-        this._propagateChange();
```

Comment:
I dont think you want to remove it from here. You need to refactor it a bit.

 - When you click on the checkbox (fine you remove it as we also use `ngModelChange` to propagate the event)
 - If you dont have checkboxes and you click just on the item, then on `onOptionClicked` and in this case you removed the EventEmitter but here you want it

Reply:
@fkolar 
On the second point, I am not sure I understood. We don't have any component in multi-combobox that doesn't have check box except one but that also emitting values as I have checked.